.travis.yml

```diff
@@ -4,31 +4,110 @@ rust:
   - nightly
 os:
   - linux
-dist: trusty
+dist: xenial
 
 matrix:
   include:
     - name: "Desktop build"
       addons:
         apt:
           packages:
-            - libpulse-dev
+            - libasound2
+            - libasyncns0
+            - libbz2-1.0
+            - libcairo2
+            - libcairo-gobject2
+            - libdatrie1
+            - libdbus-1-3
+            - libdrm2
+            - libegl1-mesa
+            - libexpat1
+            - libfaad2
+            - libflac8
+            - libfontconfig1
+            - libfreetype6
+            - libgbm1
+            - libgcrypt20
+            - libgl1-mesa-glx
+            - libglapi-mesa
+            - libglib2.0-0
+            - libgmp10
+            - libgnutls30
+            - libgpg-error0
+            - libgraphite2-3
+            - libharfbuzz0b
+            - libhogweed4
+            - libicu55
+            - libidn11
+            - libjpeg-turbo8
+            - libjson-c2
+            - liblzma5
+            - libmms0
+            - libmpg123-0
+            - libnettle6
+            - libogg0
+            - libopus0
+            - libp11-kit0
+            - libpango-1.0-0
+            - libpangocairo-1.0-0
+            - libpangoft2-1.0-0
+            - libpcre3
+            - libpixman-1-0
+            - libpng12-0
+            - libpulse0
+            - librtmp1
+            - libselinux1
+            - libsndfile1
+            - libsoup2.4-1
+            - libsqlite3-0
+            - libstdc++6
+            - libtasn1-6
+            - libthai0
+            - libtheora0
+            - libtwolame0
+            - libvorbis0a
+            - libvorbisenc2
+            - libvpx3
+            - libwayland-client0
+            - libwayland-server0
+            - libwebp5
+            - libwrap0
+            - libx11-6
+            - libx11-xcb1
+            - libxau6
+            - libxcb1
+            - libxcb-dri2-0
+            - libxcb-dri3-0
+            - libxcb-glx0
+            - libxcb-present0
+            - libxcb-render0
+            - libxcb-shape0
+            - libxcb-shm0
+            - libxcb-sync1
+            - libxcb-xfixes0
+            - libxdamage1
+            - libxdmcp6
+            - libxext6
+            - libxfixes3
+            - libxml2
+            - libxrender1
+            - libxshmfence1
+            - libxxf86vm1
+            - zlib1g
             - pulseaudio
             - pulseaudio-utils
             - dbus-x11
       services:
         - pulseaudio
 
       before_install:
-        - curl -L http://servo-deps.s3.amazonaws.com/gstreamer/gstreamer-x86_64-linux-gnu.tar.gz | tar xz
+        - curl -L http://people.igalia.com/vjaquez/servo-media/gstreamer-x86_64-linux-gnu.tar.gz | tar xz
```

Comment:
we should update our tarball cc @ferjm

Reply:
Yes, indeed.

Just I wanted to check if the bundle works.